components/Header.tsx

```diff
@@ -1,20 +1,71 @@
 import NextLink from 'next/link'
```

Comment:
General thoughts: the hamburger buttons are nor rendering.

Also, the links are not responsive and are not appearing at all in smaller views.
![Screen Shot 2021-03-02 at 10 38 34 AM](https://user-images.githubusercontent.com/72828520/109698275-1b3e6a80-7b44-11eb-8659-87c242249348.png)
![Screen Shot 2021-03-02 at 10 38 16 AM](https://user-images.githubusercontent.com/72828520/109698281-1bd70100-7b44-11eb-9462-7f8506606e25.png)